Docs: Updates readme file - FastStore v2 (#2034)
vtex/faststore: README.md

- <strong>Performance</strong>: Keep yourself calm. We use strict budgets to ensure our libs are always light and fast.
- <strong>Stability</strong>: Generate your pages ahead by using a Jamstack ready toolkit.
- <strong>SEO/Analytics</strong>: Connect to your favorite analytics provider.

> - <strong>Freedom</strong>: Use our API/SDK/UI components to connect, manage and display your ecommerce using Next.JS, a powerful React framework

Author: ```suggestion
- <strong>Freedom</strong>: Use our API/SDK/UI components to connect, manage and display your ecommerce using Next.JS, a powerful React framework.
```

- <strong>Speed</strong>: Customize our starter to fit your needs. Do not waste time creating tests, thinking about architecture or infrastructure, it's all taken care of.

## Getting started

> [Check out the docs](https://faststore.dev/) to quickstart from our official starter and start creating.

Reviewer: ```suggestion
[Check out the docs](https://www.faststore.dev/docs/getting-started-overview) to quickstart from our official starter and start creating.
```

## Official Starter

> - [NextJS](https://github.com/vtex/faststore): Basic starter with a generic store.

Author: ```suggestion
- [NextJS](https://github.com/vtex-sites/starter.store):  FastStore Starter for a generic store.
```

> - [NextJS](https://github.com/vtex/faststore): Basic starter with a generic store.

Reviewer: I think makes sense to link to the starter repo.

## Learn

Check out our [Overview](https://www.faststore.dev/docs/getting-started-overview), [How-to Guides](https://www.faststore.dev/docs/building-sections/component-customization-overview), [API References](https://faststore-site-git-api-extension-development-faststore.vercel.app/docs/api-extensions) and [Glossary](https://www.faststore.dev/docs/getting-started-glossary) to deep dive in the world of building FastStores.

> ## Contributing

Reviewer: What about a link to the [CONTRIBUTING.MD](https://github.com/vtex/faststore/blob/docs/update-readme/CONTRIBUTING.MD) as well? For example:

> This is still a work in progress. However, if you are also adventurous, you can read the code and have fun! Remember to check out our  [CONTRIBUTING.MD](https://github.com/vtex/faststore/blob/docs/update-readme/CONTRIBUTING.MD).

Author: sounds good, but I think this file is out of date :/

| :------------- | :----------------------------------------------------------------------------------------------------------------- | :---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------: |

> | @faststore/ui  | Exports basic Store components. Check out our [documentation](https://www.faststore.dev/reference/ui/faststore-ui) |   [![npm version](https://badge.fury.io/js/%40faststore%2Fui.svg)](https://badge.fury.io/js/%40faststore%2Fui) [![size minzip](https://badgen.net/bundlephobia/minzip/@faststore/ui)](https://bundlephobia.com/package/@faststore/ui) [![tree shaking](https://badgen.net/bundlephobia/tree-shaking/@faststore/ui)](https://bundlephobia.com/package/@faststore/ui) [![dependency count](https://badgen.net/bundlephobia/dependency-count/@faststore/ui)](https://bundlephobia.com/package/@faststore/ui)   |

Author: ```suggestion
| @faststore/core  | Bundles FastStore source code, including the starter for FatsStore projects: starter store. It contains three sub-packages, Components, SDK, and API, which handle the starter.) |   [![npm version](https://badge.fury.io/js/%40faststore%2Fcore.svg)](https://badge.fury.io/js/%40faststore%2Fcore) [![size minzip](https://badgen.net/bundlephobia/minzip/@faststore/core)](https://bundlephobia.com/package/@faststore/core) [![tree shaking](https://badgen.net/bundlephobia/tree-shaking/@faststore/core)](https://bundlephobia.com/package/@faststore/core) [![dependency count](https://badgen.net/bundlephobia/dependency-count/@faststore/core)](https://bundlephobia.com/package/@faststore/core)   |
| @faststore/ui  | Exports basic Store components. Check out our [documentation](https://www.faststore.dev/reference/ui/faststore-ui) |   [![npm version](https://badge.fury.io/js/%40faststore%2Fui.svg)](https://badge.fury.io/js/%40faststore%2Fui) [![size minzip](https://badgen.net/bundlephobia/minzip/@faststore/ui)](https://bundlephobia.com/package/@faststore/ui) [![tree shaking](https://badgen.net/bundlephobia/tree-shaking/@faststore/ui)](https://bundlephobia.com/package/@faststore/ui) [![dependency count](https://badgen.net/bundlephobia/dependency-count/@faststore/ui)](https://bundlephobia.com/package/@faststore/ui)   |
```

| @faststore/sdk | Exports basic logic hooks for creating your custom ecommerce                                                       | [![npm version](https://badge.fury.io/js/%40faststore%2Fsdk.svg)](https://badge.fury.io/js/%40faststore%2Fsdk) [![size minzip](https://badgen.net/bundlephobia/minzip/@faststore/sdk)](https://bundlephobia.com/package/@faststore/sdk) [![tree shaking](https://badgen.net/bundlephobia/tree-shaking/@faststore/sdk)](https://bundlephobia.com/package/@faststore/sdk) [![tree shaking](https://badgen.net/bundlephobia/dependency-count/@faststore/sdk)](https://bundlephobia.com/package/@faststore/sdk) |

> | @faststore/api | Connect your site to your ecommerce provider                                                                       |                                                                                                                                                                                             [![npm version](https://badge.fury.io/js/%40faststore%2Fapi.svg)](https://bundlephobia.com/package/@faststore/api)                                                                                                                                                                                              |

Author: Maybe we should add `faststore/core` here too.

Reviewer: Agreed :)

| @faststore/core  | Bundles FastStore source code, including the starter for FatsStore projects: starter store. It contains three sub-packages, Components, SDK, and API, which handle the starter.) |   [![npm version](https://badge.fury.io/js/%40faststore%2Fcore.svg)](https://badge.fury.io/js/%40faststore%2Fcore) [![size minzip](https://badgen.net/bundlephobia/minzip/@faststore/core)](https://bundlephobia.com/package/@faststore/core) [![tree shaking](https://badgen.net/bundlephobia/tree-shaking/@faststore/core)](https://bundlephobia.com/package/@faststore/core) [![dependency count](https://badgen.net/bundlephobia/dependency-count/@faststore/core)](https://bundlephobia.com/package/@faststore/core)   |

> | @faststore/ui  | Exports basic Store components. Check out our [documentation](https://www.faststore.dev/reference/ui/faststore-ui) |   [![npm version](https://badge.fury.io/js/%40faststore%2Fui.svg)](https://badge.fury.io/js/%40faststore%2Fui) [![size minzip](https://badgen.net/bundlephobia/minzip/@faststore/ui)](https://bundlephobia.com/package/@faststore/ui) [![tree shaking](https://badgen.net/bundlephobia/tree-shaking/@faststore/ui)](https://bundlephobia.com/package/@faststore/ui) [![dependency count](https://badgen.net/bundlephobia/dependency-count/@faststore/ui)](https://bundlephobia.com/package/@faststore/ui)   |

Author: ```suggestion
| @faststore/ui  | Exports basic Store components. Check out our [documentation](https://www.faststore.dev/components) |   [![npm version](https://badge.fury.io/js/%40faststore%2Fui.svg)](https://badge.fury.io/js/%40faststore%2Fui) [![size minzip](https://badgen.net/bundlephobia/minzip/@faststore/ui)](https://bundlephobia.com/package/@faststore/ui) [![tree shaking](https://badgen.net/bundlephobia/tree-shaking/@faststore/ui)](https://bundlephobia.com/package/@faststore/ui) [![dependency count](https://badgen.net/bundlephobia/dependency-count/@faststore/ui)](https://bundlephobia.com/package/@faststore/ui)   |
```
